nomad/grapl-core.nomad

```diff
@@ -707,7 +711,7 @@ job "grapl-core" {
         AWS_REGION                  = var.aws_region
         GRAPL_AWS_ACCESS_KEY_ID     = var.aws_access_key_id
         GRAPL_AWS_ACCESS_KEY_SECRET = var.aws_access_key_secret
-        GRAPL_AWS_ENDPOINT          = var.aws_endpoint
+        GRAPL_AWS_ENDPOINT          = local.aws_endpoint
```

Comment:
This was the only aws endpoint that was missed in @wimax-grapl 's earlier PR. This will allow the service to run correctly